views/company-number.html

```diff
@@ -0,0 +1,91 @@
+{% extends "layout.html" %}
+
+{% block pageTitle %}
+  {% if errorList.length > 0 %}
+    Error:
+  {% endif %}
+  Tell us your details
+{% endblock %}
+
+{% block beforeContent %}
```

Comment:
have a look at extensions-web, i think this might be in layout.html so it can be shared between pages.